app/Resources/views/index.html.twig

```diff
@@ -9,8 +9,9 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>mozilla.ch</title>
-    <meta name="description" content="">
+    <title>Mozilla Switzerland</title>
+    <meta name="description" content="Mozilla Switzerland is part of the global community of Mozilla Foundation, founded to manifest the state of Mozilla community in Switzerland. Internet By The People Internet ...">
+    <meta name="keywords" content="mozilla, mozillaswitzerland, firefox, india, open source, community, l10n, open, foss, software">
```

Comment:
I don't think there are any good reasons to set the keyword meta tag thee days. If search engines respect it, they use it to filter scam sites. See also Issue #34.